Dockerfile

```
@@ -1,49 +1,9 @@
-FROM postgres:14.2-alpine AS base
+FROM quay.io/ukhomeofficedigital/hocs-base-image
 
-ENV USER hocs
-ENV USER_ID 1000
-ENV GROUP hocs
-ENV NAME hocs-toolbox
-ENV PGDATA /app/data
-ENV AWS_CLI_VERSION 1.16.207
+USER root
 
-WORKDIR /app
+RUN apk add --no-cache postgresql-client kubectl aws-cli ca-certificates curl less
```

```suggestion
RUN apk add --no-cache postgresql-client kubectl aws-cli ca-certificates curl
```
Do we need `less`? It comes with busybox iirc

---

I don't know, it had Groff on there too, no idea.